supporter-product-data/src/main/scala/com/gu/services/DynamoDBService.scala

```diff
@@ -20,21 +24,30 @@ class DynamoDBService(client: DynamoDbAsyncClient, tableName: String) {
 
   def writeItem(item: SupporterRatePlanItem)(implicit executionContext: ExecutionContext) = {
     val beneficiaryIdentityId = item.gifteeIdentityId.getOrElse(item.identityId)
+
+    // Dynamo will delete expired subs at the start of the day, whereas the subscription actually lasts until the end of the day
```

Comment:
agree with adding a day.  although zuora and dates are always a fun topic - in zuora's mind it doesn't actually last until the end of the day, all dates in zuora are 00:00 midnight at the start of the day.  So they are actually out of date by the time the renewal amendment is created.